0001-Test-CASE-session-estabslishement-with-all-clusters-.patch

```diff
@@ -0,0 +1,279 @@
+From 7b16b4c67e172e4e8543a175c593d7dbe5532839 Mon Sep 17 00:00:00 2001
```

Comment:
Not sure you meant to check this one in....

Reply:
Not at all! I was pushing up changes quickly to see if CI would fail. I'll remove!